bin/BUILD.bazel

```diff
@@ -0,0 +1,5 @@
+filegroup (
```

Comment:
Should this file be under istio/mixer so that remote adapter devs can easily discovery it. Remote adapter devs will have to look for other dependencies too that live inside istio/mixer ?

Reply:
i'm open to moving this around once we get everything working. for now, i'd like to leave it here, just to keep things simple (easy to reason about run location, etc.).